elescope.el

```diff
@@ -100,16 +108,18 @@ by `run-at-time'."
               (not (seq-contains path ?/))
               (equal path (alist-get 'no-results elescope--strings)))
     (let* ((url (format "https://github.com/%s" path))
-           (name (cadr (split-string path "/")))
+	   (name (if elescope-use-namespace path (cadr (split-string path "/"))))
            (destination (expand-file-name name elescope-root-folder))
            (command (format
                      "git clone --depth=%s %s %s"
                      elescope-clone-depth
                      url
                      destination)))
-      (if (eql 0 (shell-command command))
-          (find-file destination)
-        (user-error "Something went wrong whilst cloning the project")))))
+      (if (file-directory-p destination)
```

Comment:
I believe the destination will always be a directory, do we need these lines?

Reply:
When the project has not be downloaded before and the command goes error, directory will not existed.